src/doc.d

```diff
@@ -1122,7 +1126,7 @@ extern (C++) void toDocBuffer(Dsymbol s, OutBuffer* buf, Scope* sc)
             // emit constraints if declaration is a templated declaration
             if (td && td.constraint)
             {
-                buf.writestring(" if (");
+                buf.writestring("$(DDOC_CONSTRAINT ");
```

Comment:
This looks like the first DDoc macro to default to a span element. Maybe it's better for backwards compatibility to default to `$0` and instead add `DDOC_CONSTRAINT=<span class="constraint">$0</span>` to std.ddoc.